R/utils.R

```diff
@@ -204,14 +206,22 @@ h_split_control <- function(control, ...) {
 #' Allowed are "L-BFGS-B", "BFGS", "CG"(using [stats::optim()] with corresponding method) and
 #' "nlminb"(using [stats::nlminb()]).
 #'
+#' @return named (`list`)\cr of optimizers created by [h_optimizer_fun()].
```

Comment:
```suggestion
#' @return Named `list` of optimizers created by [h_optimizer_fun()].
```
note that we only use this parentheses plus cr thing in the params, not in return

Reply:
got it!